Formula/q/qtbase.rb

```diff
@@ -0,0 +1,258 @@
+class Qtbase < Formula
```

Comment:
Re-adding comment here for visibility as I switched from `qt-base` to `qtbase` to align with repo name and avoid odd naming when we hit `qt5compat`.

Still open to discussion on best naming here.

Listing other repositories from Repology:

| Repository | Qt Base | Qt 5 Compat |
| --- | --- | --- |
| Alpine | `qt6-base` | `qt6-5compat` |
| Arch | `qt6-base` | `qt6-5compat` |
| Debian | `qt6-base` | `qt6-5compat` |
| Fedora | `qt6-qtbase` | `qt6-qt5compat` |
| Void | `qt6-base` | `qt6-qt5compat` |
| Gentoo | `dev-qt/qtbase` | `dev-qt/qt5compat` |
| Nixpkgs | `qtbase` | `qt5compat` |
| Guix | `qtbase` | `qt5compat` |

Other notes:
* Repology currently doesn't detect `qt-5compat`